docs/examples/extensions/dashboard-section/js/global-prices.js

```diff
@@ -0,0 +1,44 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import moment from 'moment';
+
+/**
+ * WooCommerce dependencies
+ */
+import { Card, Chart } from '@woocommerce/components';
+import { formatCurrency } from '@woocommerce/currency';
+
+const data = [];
+
+for ( let i = 1; i <= 20; i++ ) {
+	const today = moment().startOf( 'day' );
+	const date = today.subtract( i, 'days' );
```

Comment:
Not a big deal, but considering we are setting the hour to 00:00:00 below, there is no need to use `startOf( 'day' )` here. I think you can directly initialize the date like this:
```suggestion
	const date = moment().subtract( i, 'days' );
```

Reply:
Indeed, good call.